Manifest.toml

```diff
@@ -0,0 +1,96 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Generally speaking, Manifest.toml files shouldn't be committed. Please `git rm` this and instead add `Manifest.toml` to the .gitignore file.